monkey/monkey_island/cc/ui/src/components/report-components/security/BreachedServers.js

```diff
@@ -34,14 +35,20 @@ class BreachedServersComponent extends React.Component {
     let defaultPageSize = this.props.data.length > pageSize ? pageSize : this.props.data.length;
     let showPagination = this.props.data.length > pageSize;
     return (
-      <div className="data-table-container">
-        <ReactTable
-          columns={columns}
-          data={this.props.data}
-          showPagination={showPagination}
-          defaultPageSize={defaultPageSize}
-        />
-      </div>
+      <>
+        <p>
+          The Monkey successfully breached <span
+          className="label label-danger">{this.props.data.length}</span> {Pluralize('machines', this.props.data.length)}:
```

Comment:
Does pluralize work this way? Shouldn't you provide `machine` here?

Reply:
Fixed